tbx/static_src/sass/components/_footer.scss

```diff
@@ -91,4 +91,55 @@
             filter: none;
         }
     }
+
+    &__carbon-impact-container {
+        background-color: var(--color--footer-background);
+        border-top: 1px solid $color--grey-80;
+    }
+
+    &__carbon-impact {
+        display: flex;
+        padding: $spacer-mini-plus 0;
+        gap: $spacer-mini-plus;
+        width: 100%;
+
+        &--mobile {
```

Comment:
we should be able to simplify this by making the mobile and desktop text the same.